src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -4,22 +4,30 @@
  */
 namespace Codeception\Lib\Driver;
 
-use Facebook\FacebookSession;
-use Facebook\FacebookRequest;
+use Facebook\Facebook as Facebook_SDK;
```

Comment:
Why not just FacebookSDK? That looks cleaner to me. It is not common to use underscores in class names (unless they come from a pre-namespace era).